src/sentry/integrations/github/webhook.py

```diff
@@ -876,6 +876,12 @@ class CheckRunEventWebhook(GitHubWebhook):
     https://docs.github.com/en/webhooks/webhook-events-and-payloads#check_run
     """
 
+    # XXX: We will streamline this approach to other webhook handlers in the future.
+    # Add new webhook event processors here.
+    WEBHOOK_EVENT_PROCESSORS = [
```

Comment:
From the GitHub webhook handler, we don't care if there's going to be some preprocessing and scheduling of tasks (the function below is named `preprocess` rather than `processor`.

From the handler's perspective, we're passing the work to something that will process the work.